families/families.js

```diff
@@ -8,35 +9,69 @@ const logoutButton = document.getElementById('logout');
 logoutButton.addEventListener('click', () => {
     logout();
 });
+// const familiesList = document.getElementById('families-list');
 
-function displayFamilies() {
+async function displayFamilies() {
+    familiesEl.textContent = '';
     // fetch families from supabase
-
+    const families = await getFamilies();
+    console.log(families);
```

Comment:
Remember to pass parameters to functions that use parameters in the code, I'll mark the example where you call displayFamilies with a parameter of newFamilies.